src/core/handlers/RequestHandler.ts

```diff
@@ -194,11 +196,11 @@ export abstract class RequestHandler<
     this.isUsed = true
 
     const parsedResult = await this.parse({
-      request: mainRequestRef.clone(),
+      request: args.request,
```

Comment:
No more cloning of the intercepted request for the internal phases. Each request handler class must clone the given request by itself if it intends to read its body.